lib/marked.js

```diff
@@ -1355,6 +1355,38 @@ function splitCells(tableRow, count) {
   return cells;
 }
 
+// Remove trailing 'c's. Equivalent to str.replace(/c*$/, '').
+// /c*$/ is vulnerable to REDOS.
+// invert: Remove suffix of non-c chars instead. Default false.
+function rtrim(str, c, invert) {
+  if (typeof invert === 'undefined' || !invert) {
```

Comment:
Is this `if` statement necessary?

Reply:
It appears not. Too many years of C have addled my brain.